content/en/logs/archives/reloading/s3.md

```diff
@@ -0,0 +1,151 @@
+            ]
+        }
+    ]
+}
+```
+
+## Updating Legacy S3 Archives for Reloading
+
+Originally, Datadog's AWS S3 archiving feature depended on your S3 bucket policy to grant appropriate write access to your S3 bucket (this is the industry standard for S3 log archival). But for added scurity, Datadog now writes to your AWS S3 log archives using the IAM Role in your AWS account that you configured for [your AWS integration][2]. 
+
+Datadog only supports reloading from archives that have been configured to use role delegation. Once you have modified your Datadog IAM role to include the IAM policy above, simply edit your archive in the Log Configuration page, select which AWS Account + Role combination you wish to write and reload your archives, and then save your archive. 
+
+## Starting an Archive Reload Job
+
+To start reloading log events from your S3 archives, go to the [Historical Views tab][4] of the [Logs Configuration page][5] in your Datadog account. Select [the "Reload from Archives" button][6]. 
+
+{{< img src="logs/archives/log_historical_views.png" alt="Reload from Archives" responsive="true" style="width:75%;">}}
+
+Complete the reload form and select "Reload from Archives" to initiate the reload job. The reloaded results will go to their own special index called a "Historical View" and will have all existing facets applied. If you wish to reload from an archive that no longer exists, [follow these instructions to create that archive again][1].
+
+{{< img src="logs/archives/log_reload_from_archives.png" alt="Reload from Archives" responsive="true" style="width:75%;">}}
+
+Once you have started a reload job, the job will appear as "pending" in [the Historical Views tab][4]. When it completes it will appear as "active" and will link you to the log explorer where you can find the reloaded log events.
+
+How long a reload job takes to completed depends on the volume of data that exists in the archive for the time-range queried on, and on the count of log events that match the reload query. Reloads over large archives or large time periods can take hours to complete.
+
+## Private Beta Limitations
+
+While Archive Reloading is in private beta, the following limitations apply:
+
+1. You can only reload up to 300 Million log events with one reload job.
+2. You can only reload archived content that is older than 24 hours. 
+
+## Reload Query Rules
+
+Reload query filters can include any of the following arguments:
+
+* `service`
+* `source`
+* `status`
+* `host`
+* any attribute beginning with `@`
+* raw-text searches (which are applied on the `message` attribute)
+
+The following rules currently apply to what reload queries are supported:
+
+1. Reloading by tags is not yet supported.
+2. Use of wildcards is supported.
+3. Special characters (`+ - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /`) [must be properly escaped][7].
+4. Querying by `service` is required.
+
+### Reload Query Examples
```

Comment:
Not sure examples should be in a specific section, we should include that in the previous one.